pkg/operator/encryptionprovider/encryptionprovider_test.go

```diff
@@ -33,29 +36,30 @@ func TestEncryptionProvider(t *testing.T) {
 		expectedEncryptedGRs []schema.GroupResource
 	}{
 		{
-			name:                 "encryption off, default GRs returned",
-			defaultEncryptedGRs:  defaultGRs,
-			expectedEncryptedGRs: defaultGRs,
+			// this is default in 4.7
+			name:                 "encryption off, only OAS GRs are returned",
+			defaultEncryptedGRs:  allGRs,
```

Comment:
where are defaultEncryptedGRs even used?

Reply:
`defaultEncryptedGRs` is actually `allEncryptedGRs` - this list will be reduced once we remove the annotations. So on an upgrade 4.6 to 4.7 it will be reduced to GRs hosted by OAS